studio/pages/project/[ref]/api/index.tsx

```diff
@@ -64,14 +64,14 @@ export default observer(PageConfig)
 
 const DEFAULT_KEY = { name: 'hide', key: 'SUPABASE_KEY' }
 
-const DocView: FC<any> = observer(({}) => {
+const DocView: FC<any> = observer(({ }) => {
   const PageState: any = useContext(PageContext)
   const router = useRouter()
   const [selectedLang, setSelectedLang] = useState<any>('js')
   const [showApiKey, setShowApiKey] = useState<any>(DEFAULT_KEY)
 
-  const { data, error }: any = useSWR(`${API_URL}/props/project/${PageState.projectRef}/api`, get)
-  const API_KEY = data?.autoApiService?.defaultApiKey
+  const { data, error } = useProjectSettingsQuery({ projectRef: PageState.projectRef as string })
+  const API_KEY = data?.autoApiService?.defaultApiKey ?? "-"
```

Comment:
```suggestion
  const API_KEY = data?.autoApiService?.defaultApiKey
```

Keeping this the same as before

Reply:
I added this because Typescript was throwing a warning